Loading libDDEve in pyROOT fails with Error

Using the  FCCSW v0.8 config file on lxplus, it is possible to load libDDEve from  ROOT (`root [0] gSystem->Load("libDDEve")`) but not from pyROOT, which affects the Visualization package:

```
~$ ./run python Visualization/python/eventDisplay.py 
cling::DynamicLibraryManager::loadLibrary(): dlopen: cannot load any more object with static TLS
Traceback (most recent call last):
  File "Visualization/python/eventDisplay.py", line 6, in <module>
    assert -1 != ROOT.gSystem.Load("libDDEve"), "Could not load the Eve library! Check that libDDEve.so is in your path."
AssertionError: Could not load the Eve library! Check that libDDEve.so is in your path.
``` 

This issue might be fixed with the new release or with switching the script from python to a root macro.

--- a/Visualization/python/eventDisplay.py
+++ b/Visualization/python/eventDisplay.py
@@ -3,6 +3,18 @@ import os
 import ROOT
 ROOT.PyConfig.IgnoreCommandLineOptions = True
 import argparse
+
+
+# fix from DDG4 for TLS error on lxplus:
+# Try to load libglapi to avoid issues with TLS Static
+# Turn off all errors from ROOT about the library missing
+orgLevel = ROOT.gErrorIgnoreLevel
+ROOT.gErrorIgnoreLevel=6000
+ROOT.gSystem.Load("libglapi")
+ROOT.gSystem.Load("libQtOpenGL")
+ROOT.gErrorIgnoreLevel=orgLevel
+
+
 assert -1 != ROOT.gSystem.Load("libDDEve"), "Could not load the Eve library! Check that libDDEve.so is in your path."
 from ROOT import DD4hep
 

